### ISSUE TITLE
Drift Correction stops working

### ISSUE THREAD

**Issue by PJH-debug** [ExistentialAudio/BlackHole#322]
I have a MacBook Air using 11.2.1. I have a Focusrite interface into GarageBand and I am adding live vocal and guitar and recording video using Quicktime. This has worked fine in the past, but now it works for about 2 minutes, then I get a loud buzzing sound, and my guitar and voice sound out of tune, like they are through a phlanger. If I uncheck drift correction from my Multi-output device for the Blackhole setting, then check it again, it seems to work again for a short while, then the same happens. I have tried uninstalling black hole and re-installing but it is no different. I really need to record a video in the next week or so but am now unable to, it is driving me mad. Any help please??

**Comment by devinroth** [comment]
Upload a screenshot.

Devin

**Comment by PJH-debug** [comment]
![Screenshot 2021-02-27 at 16 52 48](https://user-images.githubusercontent.com/78421185/109393965-768e1400-791c-11eb-9cc6-d821571c178b.png)

**Comment by PJH-debug** [comment]
I originally used the 16 channel version, which workedOK, until it stopped. Looking at the web site, I tried to use the 2 channel, thinking that was my problem, but it made no difference.

**Comment by PJH-debug** [comment]
When I record audio, ie Focusrite into GarageBand I don’t need the multi-output and everything is always fine. When I record video I need to route from GB to QuickTime so that is when I use it, and gave the issue. I tried to add a video of the buzzing sound but the file was too big.

Sent from my iPhone

On 27 Feb 2021, at 17:08, Devin Roth <notifications@github.com> wrote:

﻿

Does it work normally without using a multi output?

Devin

**Comment by devinroth** [comment]
I mean BH directly to QT without a multi output.

Devin

**Comment by PJH-debug** [comment]
I didn’t realise I could do that?

Sent from my iPhone

On 27 Feb 2021, at 17:15, Devin Roth <notifications@github.com> wrote:

﻿

I mean BH directly to QT without a multi output.

Devin

**Comment by PJH-debug** [comment]
I can route the output of GB to QT just using BlackHole but I cannot monitor the sound through headphones to hear what I am doing.

**Comment by devinroth** [comment]
When I get a chance I’ll try to replicate it.

Devin

**Comment by PJH-debug** [comment]
Hi Devin, today for some reason I am not having the same issues, I don’t really know why as I don’t believe I have changed anything. At one point I unplugged the Mac to use downstairs, when I plugged everything back in, I couldn’t get any sound at all going through to QuickTime. After several hours of uninstalling, reinstalling, checking every possible option for input and output, I saw that the volume on the Blackhole interface in midi settings was almost zero. I turned it up and all is well again, but how it happened, as I didn’t know it existed before this, is a complete mystery. However, at least I learn a bit more every day 😂
